packages/webpack-plugin/src/index.js

```diff
@@ -146,17 +146,66 @@ class StylexPlugin {
           },
         );
       } else {
+        // We'll emit an asset ourselves. This comes with some complications in from Webpack.
+        // If the filename contains replacement tokens, like [contenthash], we need to
+        // process those tokens ourselves. Webpack does provide a way to reuse the configured
+        // hashing functions. We'll take advantage of that to process tokens.
+        const getContentHash = (source) => {
+          const { outputOptions } = compilation;
+          const { hashDigest, hashDigestLength, hashFunction, hashSalt } =
+            outputOptions;
+          const hash = compiler.webpack.util.createHash(hashFunction);
+
+          if (hashSalt) {
+            hash.update(hashSalt);
+          }
+
+          hash.update(source);
+
+          const fullContentHash = hash.digest(hashDigest);
+
+          return fullContentHash.toString().slice(0, hashDigestLength);
+        };
         // Consume collected rules and emit the stylex CSS asset
-        compilation.hooks.additionalAssets.tap(PLUGIN_NAME, () => {
-          try {
-            const collectedCSS = getStyleXRules();
-            if (collectedCSS) {
-              compilation.emitAsset(this.filename, new RawSource(collectedCSS));
+        compilation.hooks.processAssets.tap(
```

Comment:
`additionalAssets` was deprecated in favor of `processAssets` with the correct stage. They perform the same though.